xendit-java-lib/src/main/java/com/xendit/model/DisbursementClient.java

```diff
@@ -19,16 +19,25 @@ public Xendit.Option getOpt() {
     return opt;
```

Comment:
@hglohika can we make sure that all methods that are available on `DisbursementIDR` is with `DisbursementPH` as well?

Reply:
f8ab554fe61f23985a86e904b725d0a6fe209d99